testing_and_setup/compass/ocean/global_ocean/QU240/init/config_base_mesh.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<config case="base_mesh">
+
+	<add_executable source="mesh_converter" dest="MpasMeshConverter.x"/>
+
+	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py" dest="build_mesh.py"/>
+	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/jigsaw_driver.m" dest="jigsaw_driver.m"/>
+	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/triangle_jigsaw_to_netcdf.py" dest="triangle_jigsaw_to_netcdf.py"/>
+	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/inject_bathymetry.py" dest="inject_bathymetry.py"/>
+	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/inject_meshDensity.py" dest="inject_meshDensity.py"/>
+	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/mesh_definition_tools.py" dest="mesh_definition_tools.py"/>
+	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/global_ocean/QU240/init/define_base_mesh.py" dest="define_base_mesh.py"/>
+	<add_link source_path="mpas_tools" source="visualization/paraview_vtk_field_extractor/paraview_vtk_field_extractor.py" dest="paraview_vtk_field_extractor.py"/>
+	<add_link source_path="jigsaw-geo-matlab" source="." dest="jigsaw-geo-matlab"/>
+	<add_link source_path="jigsaw-geo-matlab" source="jigsaw/geo/topo.msh" dest="topo.msh"/>
+
+	<run_script name="run.py">
+		<step executable="./build_mesh.py">
+		</step>
+	</run_script>
+</config>
```

Comment:
It seems like this entire script is a complete copy.  Perhaps this is ok but it may be worth only having one master copy of this type of code around for clarity.

Reply:
Done.